index.js

```diff
@@ -8,9 +9,24 @@ module.exports = {
   included(...args) {
     this._super.included.apply(this, args);
 
+    const checker = new VersionChecker(this.project);
+    const ember = checker.forEmber();
+
+    this.hasNativeOnModifier = ember.gte('3.11.0-beta.1');
+
     this.hasEventHelpers = Boolean(
       this.project.findAddonByName('ember-event-helpers')
     );
+
+    if (
+      this.hasNativeOnModifier &&
+      this.hasEventHelpers &&
+      this.parent === this.project
+    ) {
+      this.ui.writeDeprecateLine(
+        'ember-on-modifier is no longer needed in your project. It can be removed from package.json.'
```

Comment:
Can you add another case for when the event helpers are not installed, but the native `{{on}}` modifier is available nonetheless?

Something like:

```
The `{{on}}` modifier is available natively since Ember 3.11.0-beta. You can remove `ember-on-modifier` from your `package.json`.
If you use the `(prevent-default)` helper, please install `ember-event-helpers`.
```

Reply:
I like this messaging much better. Should be all set.